src/Cli/dotnet/commands/dotnet-workload/install/MsiInstallerBase.cs

```diff
@@ -418,7 +418,7 @@ protected string GetMsiLogName(WorkloadPackRecord record, InstallAction action)
         /// Get a list of all MSI based SDK installations that match the current host architecture.
         /// </summary>
         /// <returns>A collection of all the installed SDKs. The collection may be empty if no installed versions are found.</returns>
-        protected IEnumerable<string> GetInstalledSdkVersions()
+        public static IEnumerable<string> GetInstalledSdkVersions()
```

Comment:
Any reason for changing the access modifier to public? Since you're moving from an instance to a static method, `internal` might be a better option.